<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddObjects</class>
 <widget class="QDialog" name="AddObjects">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adicionar Objeto</string>
  </property>
  <widget class="QTabWidget" name="Bspline">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>321</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="tabWidgetPage1">
    <attribute name="title">
     <string>Ponto</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>281</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Coordenadas do Ponto</string>
     </property>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>X</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>30</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Y</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SpinBoxPointX">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>9999.989999999999782</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SpinBoxPointY">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>9999.989999999999782</double>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButtonPoint">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>260</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Adicionar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineNamePoint">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nome:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>260</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage2">
    <attribute name="title">
     <string>Linha</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Coordenadas Finais</string>
     </property>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>X2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Y2</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SpinBoxLineX2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>9999.989999999999782</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SpinBoxLineY2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>9999.989999999999782</double>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="title">
      <string>Coordenadas Iniciais</string>
     </property>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>X1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Y1</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SpinBoxLineX1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>9999.989999999999782</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SpinBoxLineY1">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>9999.989999999999782</double>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButtonLine">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>260</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Adicionar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nome:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineNameLine">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCancel_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>260</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWidgetPage3">
    <attribute name="title">
     <string>Poligono</string>
    </attribute>
    <widget class="QPushButton" name="pushButtonPolygon">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>260</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Adicionar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineNamePolygon">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nome:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCancel_3">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>260</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>191</width>
       <height>171</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="SpinBoxPolygonX">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>80</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="SpinBoxPolygonY">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>130</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>130</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAddCoordPolygon">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>180</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Adicionar Coordenadas</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDeleteRow">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>220</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Bezier">
    <attribute name="title">
     <string>Bezier</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>301</width>
       <height>181</height>
      </rect>
     </property>
     <property name="title">
      <string>Coordenadas Bezier</string>
     </property>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>X1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Y1</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SpinBoxBezierX1">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>9999.989999999999782</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SpinBoxBezierY1">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>9999.989999999999782</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SpinBoxBezierY2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>9999.989999999999782</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SpinBoxBezierX2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>60</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>9999.989999999999782</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>X2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>60</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Y2</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SpinBoxBezierY3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>100</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>9999.989999999999782</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SpinBoxBezierX3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>100</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>9999.989999999999782</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_33">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>100</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>X3</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_34">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Y3</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SpinBoxBezierY4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>140</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>9999.989999999999782</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="SpinBoxBezierX4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>140</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <double>9999.989999999999782</double>
      </property>
     </widget>
     <widget class="QLabel" name="label_35">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>140</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>X4</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_36">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Y4</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButtonBezier">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>250</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Adicionar</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineNameBezier">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nome:</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>B-Spline</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidgetBspline_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>191</width>
       <height>171</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineNameBspline_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>70</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonDeleteRowBspline_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>200</y>
       <width>51</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAddCoordBspline_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>170</y>
       <width>171</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Adicionar Coodernadas</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="SpinBoxBsplineY_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>120</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonBspline_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>250</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Adicionar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonCancel_9">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>250</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="SpinBoxBsplineX_2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>70</y>
       <width>71</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>9999.989999999999782</double>
     </property>
    </widget>
    <widget class="QLabel" name="label_57">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nome:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
